packages/date-picker/src/DatePicker/DatePicker.tsx

```diff
@@ -1,11 +1,8 @@
 import React, { RefObject, useEffect, useRef, useState } from "react"
 import dateStart from "@kaizen/component-library/icons/date-start.icon.svg"
-import "react-day-picker/lib/style.css"
+import "react-day-picker/dist/style.css"
```

Comment:
This should belong in `Calendar`.

Looking at the styles though, if we are overriding certain default styles with our own, we might as well just completely style it ourselves and not import this (we can look at this outside of this PR).